lokikit/commands.py

```
@@ -209,6 +209,39 @@ def setup_command(ctx):
+                        "folderUid": "",
+                        "type": "file",
+                        "disableDeletion": False,
+                        "updateIntervalSeconds": 10,
+                        "allowUiUpdates": True,
+                        "options": {"path": os.path.join(base_dir, "dashboards")},
+                    }
+                ],
+            }
+
+            # Write dashboard provisioning config
+            with open(dashboard_provisioning_path, "w") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)